users/views/user_invitation_view.py

```diff
@@ -0,0 +1,31 @@
+from django.shortcuts import render
+from django.contrib import messages
+from django.contrib.auth.models import User
+from users.forms import UserInvitationForm
+from django.views.generic import View
+from users.models import SignupInvitation
+from django.core.mail import send_mail
+
+
+class UserInvitationView(View):
+    template_name = "registration/send_invitation.html"
+
+    def get(self, request):
+        form = UserInvitationForm()
+        return render(request, self.template_name, {'form': form})
+
+    def post(self, request):
+        form = UserInvitationForm(request.POST)
+        if form.is_valid():
+            invited = SignupInvitation()
+            invited.email_invited = form.data['email_invited']
+            invited.invited_by = User.objects.get(pk=request.user.id)
+            invited.save()
+            link = '/signup/{1}'.format(invited.hash)
+            send_mail('Brigid Mixs Invitation', 'Congrats for invitation!\nSignup in: {0}'.format(link), 'noreply@brigidmixs.org', [invited.email_invited])
```

Comment:
Nice email! 
But we need to add our domain to the link, because the email was like:
Signup in: /signup/1239doaspkdoapd

we need to do like this:
signup in: domain.com/signup/iajdsiaod

Reply:
Can do it using BASE_DIR?

Reply:
Done